### ISSUE TITLE
Provide accessible names for progress bars and meters

### ISSUE THREAD

**Issue by eelzi-ONRR** [DOI-ONRR/nrrd#2539]
**OCIO Violation:** Provide accessible names for progress bars and meters

**OCIO Notes:** This DIV (role=progressbar) does not have a mechanism that allows an accessible name value to be calculated

OCIO priority: 2.3
More info in [this sheet](https://doimspp.sharepoint.com/:x:/s/NRRD/Ea-CVbPsGDpDkDLSIe6kZS8BNeTJ7C8_M7yRS86t6VEU2g) or the [ocio/amp dashboard](https://departmentofinterior.levelaccess.us/public/reporting/prioritization_tool.php?report_id=1532)

**Instances** 

- [ ] Progress bar for initial page download (horizontal bar) - https://revenuedata.doi.gov/query-data/?dataType=Revenue#main-content

- [ ] Progress bars for compare revenue cards, nationwide revenue summary, etc. - https://revenuedata.doi.gov/explore/?dataType=Revenue&location=NF,NA&mapLevel=State&offshoreRegions=false&period=Calendar%20Year&year=2022

**Comment by eelzi-ONRR** [comment]
@greensomes 

For both of these, the alt text should be: 

"Progress bar indicating content on the page is loading"

**Comment by eelzi-ONRR** [comment]
2024 OCIO report

Instances: 14
Occurrence: 1% 
Severity: 5
Priority: 2.3

**Comment by lpgoldstein** [comment]
No longer relevant
